package.json

```diff
@@ -44,7 +44,7 @@
   },
   "devDependencies": {
     "@monaco-editor/react": "^4.4.5",
-    "@transcend-io/airgap.js-types": "^10.4.2",
+    "@transcend-io/airgap.js-types": "^10.6.0",
```

Comment:
I don't think I understand why the version jump is so large? Not sure if that makes sense.

Reply:
10.4.2 was published [2 months ago](https://github.com/transcend-io/airgap.js-types/pkgs/npm/airgap.js-types/versions). There have been 8 versions between 10.4.2 and 10.6.0 (including 6 point versions of 10.5) but no one bothered to update consent-manager-ui until now. I suppose it wasn't necessary, but this is a breaking change.

Reply:
nit: it's not a breaking change in functionality/the API of this library, it's just a big type change..

Reply:
True. It's a breaking-tsc change, but not a breaking-functionality change.

Reply:
Actually, to be fair, this (type) actually broke in [10.5.0](https://github.com/transcend-io/airgap.js-types/pull/91) with the introduction of `TCF_EU` to `ViewState`. The change from 10.5 to 10.6 is, in fact, unrelated to the fix in BottomMenu.